addons/forum-time-zones/matcher.js

```diff
@@ -0,0 +1,66 @@
+export default async function ({ addon, global, console }) {
+  // check if setting enabled
```

Comment:
Should it be "check if twelve-hour-time is enabled"?

Reply:
This comment is probably unnecessary, so don't worry about it.